youtube_dl/YoutubeDL.py

```diff
@@ -1563,8 +1569,12 @@ def process_info(self, info_dict):
                             subfile.write(sub_data)
                 except (OSError, IOError):
                     self.report_error('Cannot write subtitles file ' + sub_filename)
+                    all_subtitles_ok = False
                     return
```

Comment:
This ```return``` is introduced in a0432a1e8086e2318b122744c7afa5145abdf663. I guess failures in downloading subtitles should be non-fatal, just like thumbnails?